docs/evaluate/temporal-cloud/legacy-support.mdx

```diff
@@ -0,0 +1,160 @@
+- **Design and code reviews:** Our team works with you to review your Workflow design and code to avoid potential issues and optimize your workload to run on Temporal Cloud.
+  These sessions also focus on the optimization of your Workflow Actions and reduce your costs; they also reveal likely trade offs.
+
+## What is the Temporal Cloud support guarantee? {#support}
+
+Temporal endeavors to ensure you are successful with Temporal Cloud.
+We offer explicit guarantees for support.
+Temporal Cloud customers get break/fix support with an agreed-upon set of SLAs for prioritized issues.
+We use a ticketing system for entering, tracking, and closing these issues.
+
+If an issue occurs, the team also provides support through a dedicated Slack channel, forums, and a knowledge base.
+We offer two levels of support defined by their availability and SLAs in the following table:
+
+|                          | **Basic**                                                                                      | **Enterprise**                                                                            |
+| ------------------------ | ---------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
+| **Response times**       | P0: 1 business hour<br />P1: 4 business hours<br />P2: 1 business day<br />P3: 2 business days | P0: 30 minutes<br />P1: 1 business hour<br />P2: 4 business hours<br />P3: 1 business day |
+| **Onboarding & Success** | Development, deployment, and optimization guidance                                             | Development, deployment, and optimization guidance                                        |
+| **Channels**             | Community Slack, Email, Forums, Documentation, Knowledge Base                                  | Private Slack Channel, Email, Forums, Documentation, Knowledge Base                       |
+
+:::info BUSINESS HOUR TIMEZONES
+
+Business Hours will be specified in your contract, including one of three locations: US local time, European Central time, Australia Eastern time
+:::
+
+**Priority definitions**
+
+- **P0 - Critical** (Production impacted)
+  - The Temporal Cloud service is unavailable or degraded with a significant impact.
+- **P1 - High** (Production issue)
+  - An issue related to production workloads running on the Temporal Cloud service, or a significant project is blocked.
+- **P2 - Normal** (General issues)
+  - General Temporal Cloud service or other issues where there is no production impact, or a workaround exists to mitigate the impact.
+- **P3 - Low** (General guidance)
+  - Questions or an issue with the Temporal Cloud service that is not impacting system availability or functionality.
+
+:::note ON PAGE SERVICE
```

Comment:
Same here. Do we capitalize these admonitions?

Reply:
This is automatically capped in rendering. Putting it in caps in the mdx reflects how it will be rendered.

Reply:
Lower casing these and pushing.